### DISCUSSION TITLE
How to hide menu bar in my desktop window?

### DISCUSSION THREAD

**Discussion by miaomiaowu0428** [DioxusLabs/dioxus#3039] [Q&A]
As my title: 
![image](https://github.com/user-attachments/assets/75d11b65-1c58-474a-b1ca-83a751c1fa8b)
I want to hide this, what should I do?

**Comment by InAnYan** [comment]
And how to customize the menu bar?

**Comment by ellcs** [comment]
Just figured out that you can expand `dioxus::launch(App)` by using `LaunchBuilder`. You can disable the menu using `with_menu(None)`. As provided in the snippet below:

```rust
fn main() {
    dioxus::LaunchBuilder::new()
        .with_cfg(
            Config::default().with_menu(None).with_window(
                WindowBuilder::new()
                    .with_maximized(true)
                    .with_title("Catchy title")
                )
            )
        .launch(App);
}
```

Currently, the `doc.rs` build is not really usable to me. I ran `cargo doc` in my project and inspected the dioxus documentation there. You would find it at `$PATH_TO_YOUR_PROJECT/target/doc/dioxus_desktop/struct.Config.html#method.with_menu`.

**Reply by ellcs** [reply]
In the end, I used the approach in the discussion #635. With the advantage to not use the `dioxus-desktop` crate. Which resulted in:

```rust
fn main() {
    #[cfg(feature = "desktop")]
    fn launch_app() {
        use dioxus::desktop::tao;
        let window = tao::window::WindowBuilder::new().with_resizable(true);
        dioxus::LaunchBuilder::new().with_cfg(dioxus::desktop::Config::new().with_window(window).with_menu(None)).launch(App);
    }

    #[cfg(not(feature = "desktop"))]
    fn launch_app() {
        dioxus::launch(App);
    }

    launch_app();
}
```
